lib/reek/context/code_context.rb

```diff
@@ -137,7 +137,7 @@ def config_for(detector_class)
       #
       # @param visibility [Symbol]
       # @param names [Array<Symbol>]
-      def track_visibility(visibility, names = [])
+      def track_visibility(visibility, names: [])
```

Comment:
Let's make names a required argument since it's never called without names in the actual code.